script/prettier.ts

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env ts-node
+
+import * as Path from 'path'
+import chalk from 'chalk'
+import { spawnSync } from 'child_process'
+
+const root = Path.dirname(__dirname)
+
+const prettier = process.platform === 'win32' ? 'prettier.cmd' : 'prettier'
+const prettierPath = Path.join(root, 'node_modules', '.bin', prettier)
+
+const result = spawnSync(prettierPath, ['**/*.{md,scss}', '--list-different'], {
+  cwd: root,
+})
+
+if (result.status > 0) {
+  process.exitCode = result.status
+
+  console.log('These files are not formatted correctly:\n')
+  console.log(result.stdout.toString())
+
+  console.error(
+    chalk`{bold.green → To fix these errors, run {underline yarn lint:fix}}`
```

Comment:
This one could just be `yarn prettier --write` — that way, each command will tell you how to fix its own errors (this will also make the fixing process faster since you don’t have to run both). What do you think?

Reply:
Makes sense

Reply:
This would also mean that you need to provide any CLI options to Prettier.

Reply:
@j-f1 I'm not quite sure I understand

Reply:
@shiftkey Never mind, your solution looks good.